src/l1-resources/access-list-api-key/integ.default.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright 2023 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
```

Comment:
genuine curiousity: what is the rationale for which we put this header to some files and some other not?

Reply:
there was a github action that checked the licences, i think is not working anymore, probably we can delete the license on this one

Reply:
> there was a github action that checked the licences, i think is not working anymore

What was the github action doing? I think we should rather fix it? I believe license has a purpose and removing it might have some consequences in the future